src/PrestaShopBundle/Resources/config/services/adapter/product/_combination.yml

```diff
@@ -53,6 +53,9 @@ services:
             - '@prestashop.adapter.product.stock.repository.stock_available_repository'
             - '@prestashop.core.grid.query_builder.product_combination'
             - '@prestashop.adapter.attribute.repository.attribute_repository'
+            - '@=service("prestashop.adapter.legacy.context").getContext().shop.id'
+#            @todo; originally _PS_THEME_PROD_DIR_ was used, but it points to same dir and its not visible in tests. Can i use PROD_IMG_DIR?
+            - !php/const _PS_PROD_IMG_DIR_
```

Comment:
wdyt?
If I use PS_THEME_PROD_DIR its not visibile during tests runtime. Even if I include definces_uri.inc in autoload :thinking: 
Is it important to use theme const over prod (they are pointing to same dir)

Reply:
Well actually I couldn't find **any** reference to `_PS_THEME_PROD_DIR_` (no comment, no string, no nothing)
So I'd say you don't have much choice 😅